fr-FR.json

```diff
@@ -854,113 +854,114 @@
-	
-	"FAQ_22_H1": "I cannot see/use most add-ons on iOS",
-	"FAQ_22_P1": "Because of restrictions in the Apple App Store terms and conditions, we have restricted the iOS application only to a particular set of add-ons.",
-	"FAQ_22_P2": "But rest assured - there will be more add-ons for iOS in the future, and we may also add an integration that will allow you to use all Stremio add-ons thanks to connecting the iOS app to the Stremio Streaming Server."
-
-}
\ No newline at end of file
+	
+	"FAQ_001_H1" : "Qu'est-ce que Stremio ?",
+	"FAQ_001_P1" : "Stremio est un logiciel de media center conçu pour le divertissement vidéo dans un souci de facilité d'utilisation. Stremio vous permet d'organiser &amp ; regardez votre propre collection de films, de séries, de chaînes et de télévision en direct",
+	"FAQ_001_P2" : "Stremio collecte le contenu qu'il vous montre à partir de votre disque local et de notre système de modules d'extension.",
+	"FAQ_001_P3" : "Il vous avertira également de la sortie de nouveaux épisodes/vidéos/films qui vous intéressent et vous fournira des recommandations de films",
+	
+	"FAQ_02_H1" : "Comment Stremio se compare-t-il à Kodi et Plex ?",
+	"FAQ_02_P1" : "Nous sommes inspirés par ces deux logiciels. Mais nous avons des objectifs différents - Stremio est spécifiquement destiné à la vidéo de divertissement, et s'efforce de devenir la solution la plus courante pour cela en étant très facile à utiliser",
+	"FAQ_02_P2" : "Stremio ne sera jamais aussi personnalisable et riche en fonctionnalités que Kodi et Plex, mais il est nettement plus facile à utiliser. Just ${1} get it and see ${2} :)",
+	
+	"FAQ_03_H1" : "Comment Stremio se compare-t-il à Netflix ?",
+	"FAQ_03_P1" : "Encore une fois, Netflix est une de nos inspirations. Nous nous efforçons d'atteindre l'expérience utilisateur de Netflix, plus de nombreux goodies ajoutés. Mais surtout, nous vous donnons la liberté de gérer votre propre bibliothèque et d'ajouter de nouvelles sources de contenu grâce à notre système d'extension",
+	"FAQ_03_P2" : "Cela rend Stremio infiniment extensible, donc quand vous utilisez Stremio, vous n'êtes jamais limité au contenu que Netflix décide de licencier. Par exemple, grâce à Stremio, vous pouvez regarder du contenu à partir de ${1} Filmon.tv ${2}, ${3} Twitch.tv${4}, ${5}YouTube${6}, ${7}Vodo${8} et même ${9}Netflix${10} lui-même !",
+	
+	"FAQ_04_H1" : "Comment Stremio se compare-t-il à Flipboard ?",
+	"FAQ_04_P1" : "On ne nous pose pas souvent cette question, sauf quand on parle à des gens de la start-up :)",
+	"FAQ_04_P2" : "Stremio est par conception un agrégateur de contenu, ce qui en fait essentiellement un Flipboard pour la vidéo. Cela est très visible dans notre section "Board", qui vous donne des catalogues et des notifications de nouveaux épisodes/nouvelles vidéos pour ce que vous suivez, tout comme Flipboard",
```

Comment:
There is a syntax error here - you must add a \ before each " in "Board"

Reply:
I just added the "adding the". It's not working.